### PR TITLE
Refactored 'No' radio button validation to resolve live issues

### PR DESCRIPTION
**Pull request by justinhyltonHO**
**What changes does this PR bring?**

🚨 **Henry observed the following Live issue error:**

Live issue error

-  ✅ I refactored the code which should resolve this error.

**Definition of Done**
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.
- [ ✅] Have you built the code locally and confirmed its working?
- [ ✅] Is the build confirmed to be passing on CI?

🧪 **Evidence of manual testing:**

#### 1a. Manual Tests
1a  Manual Tests

#### 1b. Page loads
1b  Page loads

#### 1c. 'No' radio button clicked with no data entered
1c  No radio button clicked with no data entered

#### 1d. 'No' radio button clicked with data
1d  No radio button clicked with data

#### 1e. 'Yes' radio button clicked
1e  Yes radio button clicked

### CODE REVIEW

File: src/app/garfile/departure/index.njk
Comment (ame-test) [inline src/app/garfile/departure/index.njk:177]: Question: why was the `!=` chosen over `!==`?
e.g to take `null` or `undefined` value, rather than just `undefined`.

I am just curious.

**Review by ame-test — APPROVED** [review]
Looks good, as long as you have tested it, I am ok for it to be approved.
*Two minor questions, optional - out of curiousity.

**Comment by ame-test** [comment]
What was the cause of the `undefined` error?

**Review by ame-test — CHANGES_REQUESTED** [review]
Actually, hold up --> the change requires test evidence since it is a bug from production.
* I apologise for missing that, it is my mistake.

**Comment by justinhyltonHO** [comment]
> What was the cause of the `undefined` error?
> 
>     * e.g. what cause the `cookie["getDepartureVoyage"]["departureLat"]["includes"]` value to be `undefined` or `falsey`?

**Comment by justinhyltonHO** [comment]
> > What was the cause of the `undefined` error?
> > ```
> > * e.g. what cause the `cookie["getDepartureVoyage"]["departureLat"]["includes"]` value to be `undefined` or `falsey`?
> > ```

**Comment by justinhyltonHO** [comment]
> > > What was the cause of the `undefined` error?
> > > ```
> > > * e.g. what cause the `cookie["getDepartureVoyage"]["departureLat"]["includes"]` value to be `undefined` or `falsey`?
> > > ```

When the page loads cookie.getGarDepartureVoyage().departureLat is undefined.  I reproduced the error.
0a  Pages loads with production code
0b  Production Error reproduced

File: src/app/garfile/departure/index.njk
Comment (justinhyltonHO) [reply src/app/garfile/departure/index.njk:177]: > Question: why was the `!=` chosen over `!==`? e.g to take `null` or `undefined` value, rather than just `undefined`.
> 
> I am just curious.

Strict equality fails because cookie.getGarDepartureVoyage().departureLat is type String not type undefined.

**Review by ame-test — APPROVED** [review]
Approved as testing was provided.
* Thanks for doing so.

**Comment by ame-test** [comment]
This changes is going to be merge so that it can be added to Monday's release.
* This is because the bug this change fixes is both severe and somewhat frequent - prevents users from editing any departure date and arrival date if they submitted the GAR via excel upload.
* Therefore, it is being deployed and tested to SIT and Staging so it is ready for Monday release or sooner.

**Comment by justinhyltonHO** [comment]
[like]  Justin Hylton reacted to your message:
________________________________

**Comment by justinhyltonHO** [comment]
[like]  Justin Hylton reacted to your message:
________________________________

**Pull request merged**
